proctor-common/src/main/java/com/indeed/proctor/common/model/TestDefinition.java

```diff
@@ -55,6 +56,8 @@ public class TestDefinition {
 
     private boolean enableUnitlessAllocations;
 
+    private Date lastRevisionDate;
```

Comment:
One thing to consider. Using date object will take up more memory than if we use a the boolean and calculate in proctor-builder based on "lastRevisionDate - build date > X". Can discuss with the rest of team if worth doing it this way

Reply:
agreed. boolean is better